test/validation/api/pktio/pktio.c

```diff
@@ -82,9 +82,15 @@ typedef enum {
 typedef enum {
 	TXRX_MODE_SINGLE,
 	TXRX_MODE_MULTI,
-	TXRX_MODE_MULTI_EVENT
+	TXRX_MODE_MULTI_EVENT,
+	TXRX_MODE_VECTOR,
 } txrx_mode_e;
 
+#define TXRX_FLAG(x)	      (0x1UL << (TXRX_MODE_##x))
+#define TXRX_FLAG_ISSET(f, x) (((f) >> (TXRX_MODE_##x)) & 0x1UL)
```

Comment:
This may be a matter of taste, but I'd prefer setting the enumerators directly to bit values, so that standard bitwise operations could be used. With the current style one can easily make the mistake of using the enumerator directly and causing unexpected behavior.

E.g.
```
typedef enum {
	TXRX_FLAG_SINGLE = 0x1,
 	TXRX_FLAG_MULTI = 0x2,
	TXRX_FLAG_MULTI_EVENT = 0x4,
	TXRX_FLAG_VECTOR = 0x8,
} txrx_flag_e;
```